pages/interface/components/ContentList/index.js

```diff
@@ -21,7 +21,8 @@ export default function ContentList({ contentList, pagination, paginationBasePat
             display: 'grid',
             gap: '0.5rem',
             gridTemplateColumns: 'auto 1fr',
-          }}>
+          }}
+          role={'list'}>
```

Comment:
Conforme comentários do [hitmain13](https://github.com/hitmain13):

```suggestion
          role="list">
```